packages/modals/src/styled/StyledBackdrop.ts

```diff
@@ -18,21 +16,46 @@ export interface IStyledBackdropProps {
   isAnimated?: boolean;
 }
 
+const animationStyles = () => {
+  const animationName = keyframes`
+    0% {
+      opacity: 0;
+    }
+
+    to {
+      opacity: 1;
+    }
+  `;
```

Comment:
If this is a consistent animation lets move it as a constant so it isn't being newed up during each render.

Reply:
I've removed the function call. Now, the animations should not be newed up each render. Thanks for the suggestion!